src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -508,18 +550,29 @@ export class AppRoot extends mixinBehaviors
       // An array of {id, name, dir} language objects.
       supportedLanguages: {type: Array, readonly: true},
       useKeyIfMissing: {type: Boolean},
+      accountList: {type: Array},
       serverList: {type: Array},
       selectedServerId: {type: String},
       hasManualServers: {
         type: Boolean,
         computed: '_computeHasManualServers(serverList.*)',
       },
-      digitalOceanAccountName: {type: String},
+      digitalOceanAccountName: {
```

Comment:
You are not benefitting from having an Account list.
You could remove that property and have two instead: doAccount and gcpAccount, both could be some sort of AccountEntry with name and id/cloud.
This would remove the need for all those computed properties.
!!doAccount is connected, doAccount.name is the name